src/actions/types/SourceAction.js

```diff
@@ -61,6 +61,6 @@ export type SourceAction =
     |}
   | {|
       +type: "CLOSE_TABS",
-      +urls: string[],
+      +sources: Array<Source>,
```

Comment:
@jasonLaster Can you advise on why flow doesn't like this?  I tried `Source[]` and that blew up too.

Reply:
`Source[]` should be the same... strange